libse/Paragraph.cs

```diff
@@ -114,12 +114,8 @@ public void Adjust(double factor, double adjustmentInSeconds)
         {
             if (StartTime.IsMaxTime)
                 return;
-
-            double seconds = StartTime.TimeSpan.TotalSeconds * factor + adjustmentInSeconds;
-            StartTime.TimeSpan = TimeSpan.FromSeconds(seconds);
-
-            seconds = EndTime.TimeSpan.TotalSeconds * factor + adjustmentInSeconds;
-            EndTime.TimeSpan = TimeSpan.FromSeconds(seconds);
+            StartTime.TotalSeconds *= factor + adjustmentInSeconds;
```

Comment:
*= just multiplies, so it gives StartTime.TotalSeconds = StartTime.TotalSeconds * (actor + adjustmentInSeconds)
Did you test it?